src/Fixer/ClassNotation/StaticPrivateMethodFixer.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ClassNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class StaticPrivateMethodFixer extends AbstractFixer
+{
+    /**
+     * @var array
+     */
+    private $magicNames = [
+        '__call' => true,
+        '__callstatic' => true,
```

Comment:
`__callStatic` can't be non-static, so it could be removed here.

Reply:
I'll refine this list even further, as a lot of this methods cannot be private, thank you

Reply:
I'll refine this list even further, as a lot of this methods cannot be private, thank you